### ISSUE TITLE
React Native - API Response 200 "skipped"

### ISSUE THREAD

**Issue by monkeyinsurgency** [infinitered/reactotron#884]
Responses showing as "skipped" regardless of status.  Response Headers, Request, and Request Headers all available, just not Response.

<img width="589" alt="image" src="https://user-images.githubusercontent.com/1014085/49217593-e0c65000-f3c5-11e8-9d5f-ccf8fdf87d34.png">

**Comment by chawax** [comment]
I had this "skipped" message when the service returned a plain text content while I expected JSON.

**Comment by pekq** [comment]
Yes, `skipped` means the response is empty. It's misleading. Took me awhile to figure out that it's not the actual response from the API...

**Comment by monkeyinsurgency** [comment]
I'm definitely getting a response...

![image](https://user-images.githubusercontent.com/1014085/49439726-b86d9580-f7b9-11e8-941d-6e57f2a28ccc.png)

![image](https://user-images.githubusercontent.com/1014085/49439731-c02d3a00-f7b9-11e8-827a-28f177bd5292.png)

**Comment by rmevans9** [comment]
Hmm... I personally haven't seen it show "skipped" when you get an actual response. Is this happening only for this specific endpoint? Are you using `react-native`? If so can you tell me the following versions:

react-native version?
reactotron version? (the desktop client)
reactotron-react-native version?
Are you using any other reactotron plugins?

Thanks!

**Comment by rmevans9** [comment]
All that seems up to date enough. Is this endpoint something public that I can try out? If not, can you reproduce on a different endpoint?

Also - can you show the code that makes this request and let me know which library (if any) you are using to do the network requests?

**Comment by scabezasiq** [comment]
@monkeyinsurgency, could you solve the error?

**Comment by kevinvangelder** [comment]
Could we show something like `{empty}` instead of `skipped` and make it visually different to help make this clearer? Obviously that won't help the people who are getting `skipped` unexpectedly.

**Comment by rmevans9** [comment]
@kevinvangelder Yes, we can. The default value is set here: https://github.com/infinitered/reactotron-react-native/blob/master/src/plugins/networking.ts#L90

**Comment by mayermatheus** [comment]
I am also having this problem with a request with soap protocol

**Comment by calendee** [comment]
What's confusing is this:

https://github.com/infinitered/reactotron-react-native/blob/master/src/plugins/networking.ts#L99

That seems to be where the `skipped` comes from.  However, the Reactotron UI does not show the tildes.  

If you save the response to clipboard as markdown, it looks like (not dashes added to avoid section being considered another code block):

```
### Data Received

-```json
 skipped 
-```

```

It would be much more informative and less confusing for the UI and copied markdown to show something like

`No Response in Body`.

`skipped` does not clearly indicate what the response was.

**Comment by ramosalx1504** [comment]
In my case, I use "axios" to make https requests.

When I configure the library with a very short "timeout" for the requests, all the requests result in "skipped" if the server is not able to respond before reaching the "timeout".

**Comment by rmevans9** [comment]
@ramosalx1504 this is currently expected as no body means `reactotron` is going to render "skipped"

**Comment by haroonrashid2210** [comment]
> Are you using Axios or Apisauce to make requests? This has happened to me before with those libraries when the request is blocked before being sent. (Insecure HTTP, timeouts, or even the app having internet connection issues).
> 
> Reactotron only shows the status of the request, and in this case it was blocked by your app in some way.

I am using Axios. How did you solve the problem?
